aip/general/0193.md

```diff
@@ -75,205 +72,393 @@ following:
+<em>Example message</em>: "The zone 'us-east1-a' does not have enough resources available to fulfill the request. Try a different zone, or try again later."
+   </td>
+  </tr>
+  <tr>
+   <td><code>details <em>Object (repeated)</em></code>
+   </td>
+   <td> Text
+<p>
+<em>Example</em>:  Described in the <a href="#error-details">ErrorDetails </a> section.
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+#### [ErrorDetails](https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto)
+
+Google defines a set of [standard detail payloads](https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto) for error details, which cover most common needs for API errors. Services **should** use these standard detail payloads when feasible.  
+
+Structured details with machine-readable identifiers **must** be used if users will need to write code against a specific aspect of the error. Error message strings **may** change over time; however, if an error message does not have a machine-readable identifier _in addition to_ the `code` string, changing the error message **must** be considered a backwards-incompatible change.
+
+For the purpose of this discussion, the ErrorDetails section contains three payloads: `ErrorInfo`, `LocalizedMessage`, and `HelpLink`.
+
+##### Uniqueness
+
+Each type of detail payload **must** be included at most once. For example, there **must not** be more than one <code>[BadRequest](https://github.com/googleapis/googleapis/blob/477a59d764428136ba1d857a9633c0d231de6efa/google/rpc/error_details.proto#L168-L218)</code> message in the <code>details</code>, but there <strong>may</strong> be a <code>BadRequest</code> and a <code>[PreconditionFailure](https://github.com/googleapis/googleapis/blob/6f3fcc058ff29989f6d3a71557a44b5e81b897bd/google/rpc/error_details.proto#L139-L166)</code>.
+
+##### ErrorInfo payload
+
+The <code>[ErrorInfo](https://github.com/googleapis/googleapis/blob/6f3fcc058ff29989f6d3a71557a44b5e81b897bd/google/rpc/error_details.proto#L27-L76)</code> message is the required way to send a machine-readable identifier. All error responses <strong>must</strong> include an <code>ErrorInfo</code> payload in <code>details</code>. Variable information <strong>should</strong> be included in <code>metadata</code> in <code>ErrorInfo</code> and <strong>must</strong> be included if it appears within an error message.
+
+When introducing an error that represents a failure scenario that did not previously occur for the service, the payload **must** include `ErrorInfo` and any variables found in dynamic segments of the error message **must** be present in `ErrorInfo.metadata` (See [Dynamic variables](#dynamic-variables).
+
+**Note:** `ErrorInfo` represents a special case. There **must** be exactly one `ErrorInfo`. It is required.
+
+[ErrorInfo Payload](https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto#L51)
+
+
+<table>
```

Comment:
we've avoided using HTML in AIP markdown so far, could you replace the HTML with its markdown equivalent where possible? e.g. there's a markdown equivalent for <table>, <li>, <p>, <em>, <code>. Also you shouldn't need to HTML escape quotes if you'll only use markdown.